pocket_coffea/lib/scale_factors.py

```diff
@@ -471,3 +471,29 @@ def sf_pileup_reweight(params, events, year):
     sfdown = puWeightsJSON[puName].evaluate(nPu, 'down')
 
     return sf, sfup, sfdown
+
+def sf_isr(events):
```

Comment:
better to add parton shower in the name:  in my opinion better  `sf_partonshower_isr`  than `sf_ps_isr`.

Reply:
Done